Add section 1.2.2 (#8)
jneidel/internet.org-entwicklung: main.tex

\begin{document}

\maketitle
% 1.2.2 in der Gliederung
\subsubsection{Internetverbreitung}

> In Indien leben 2015 schon 350 Millionen Menschen\autocite{slideshareIndia} welche das Internet aktiv nutzen, wovon allein 159 Millionen per Mobilgerät unterwegs sind. Obwohl Indien damit schon den 2. Platz der aktiven Internetnutzer weltweit genießt, haben lediglich 27\% der indischen Bevölkerung Zugang zum Internet\autocite{InternetCountry}.

Reviewer: ```suggestion
In Indien leben 2015 schon 350 Millionen Menschen welche das Internet aktiv nutzen\autocite{slideshareIndia}, wovon allein 159 Millionen per Mobilgerät unterwegs sind.
Obwohl Indien damit schon den 2. Platz der aktiven Internetnutzer weltweit genießt, haben lediglich 27\% der indischen Bevölkerung Zugang zum Internet\autocite{InternetCountry}.
```

\medskip 
Ein Computer ist in Indien ein Luxusgut, welches durch die rasante Entwicklung von mobilen Endgeräten für die Jugend schnell an Bedeutung verlor, während Handys mit mobilem Internet allgegenwärtig wurden.
Durch die explosionsartige Verbreitung der neuen Kommunikationsmöglichkeiten entstanden in kürzester Zeit große Gruppen im Internet, welche sich über neue Technologien, Wissenswertes und Ihre Interessen austauschten.

> Dieser rapide Aufbau von indivuellen Gruppen ist ein neues Phänomen, welches vorallem in `Entwicklungsländern' auftritt, in denen neue, und vergleichsweise preiswerte Technologien, den Zugang zum Internet ermöglichen. \textcite{empowermentThroughFacebook}

Reviewer: ```suggestion
Dieser rapide Aufbau von indivuellen Gruppen ist ein neues Phänomen, welches vorallem in Entwicklungsländern auftritt, in denen neue, und vergleichsweise preiswerte Technologien, den Zugang zum Internet ermöglichen \textcite{empowermentThroughFacebook}.
```

\medskip
Die durchschnittliche Bandbreite mobilen Internets war mit 2.8 Mb/s der durchschnittlichen Bandbreite eines Festanschlusses, von 2.3 Mb/s, überlegen \autocite{slideshareIndia}, wodurch das Interesse an einen Computer noch mehr abnahm.

> Der Preis einer Prepaid Karte mit 1 GB Datenvolumen betrug derzeit ca. 3.55\% des durchschnittlichen Monatsgehalt\autocite{broadbandAgency}, und ist durch den hohen Preis nur für städtisch lebende Inder erschwinglich, da dort das Einkommen weitaus höher ist.

Reviewer: ```suggestion
Der Preis einer Prepaid Karte mit 1 GB Datenvolumen betrug damals ca. 3.55\% des durchschnittlichen Monatsgehalt\autocite{broadbandAgency}, und ist durch den hohen Preis nur für städtisch lebende Inder erschwinglich, da dort das Einkommen weitaus höher ist.
```

> Obwohl gerademal 31,2\% der indischen Bevölkerung in städtischen Gebieten lebt, leben von den 350 Millionen aktiven Internetnutzern gerademal 61 Millionen auf dem Land\autocite{IndiaBevölkerung}. Somit benutzten 2015 gerademal 7.5 \%, auf dem Land lebende Inder, aktiv das Internet \autocite{slideshareIndia}.

Reviewer: ```suggestion
Obwohl gerademal 31,2\% der indischen Bevölkerung in städtischen Gebieten lebt, machen sie 82\% der aktiven Internetnutzer aus\autocite{IndiaBevölkerung}.
Somit benutzten 2015 gerademal 7.5 \%, auf dem Land lebende Inder, aktiv das Internet \autocite{slideshareIndia}.
```

\begin{document}

\maketitle

> \subsubsection Internetverbreitung

Reviewer: ```suggestion
% 1.2.2 in der Gliederung
\subsubsection{Internetverbreitung}
```

> In Indien gibt es 2015 schon 350 millionen Menschen\autocite{slideshareIndia} , welche das Internet aktiv nutzen, und davon sind 159 millionen per Handy im Internet unterwegs. Somit sind sie schon auf Platz 2 der aktiven Internetnutzer weltweit. So groß sich die Zahl auch anhören mag, das sind lediglich 27\% der Bevölkerung\autocite{InternetCountry}. 

Reviewer: ```suggestion
In Indien leben 2015 schon 350 Millionen Menschen\autocite{slideshareIndia}, welche das Internet aktiv nutzen, und davon sind 159 Millionen per Handy im Internet unterwegs. Somit sind sie schon auf Platz 2 der aktiven Internetnutzer weltweit. So groß sich die Zahl auch anhören mag, das sind lediglich 27\% der Bevölkerung\autocite{InternetCountry}. 
```

Reviewer: `So groß sich die Zahl auch anhören mag` vlt. ein bisschen formeller machen :smile:

> Laut einer Studie sei 2015 ein PC in Indien ein Luxusgut, welches durch die schnelle Entwicklung des Handys für die Jugend schnell die Bedeutung verlor, und Handys mit Internet in der Gesellschaft allgegenwärtig wurde. Durch die explosionsartige Verbreitung der neuen Möglichkeit mit anderen zu kommunizieren und sich auszutauschen bauten sich in kurzer Zeit große Gruppen im Internet auf, um sich über neue Technologien, Wissenswertes und Interessen auszutauschen. Dieser rapide Aufbau von Gesellschaften ist ein neues Phänomen, welches vorallem in "Entwicklungsländern" auftritt, in denen neue und vergleichsweise preiswerte Technologien den Zugang zum Internet ermöglichen.   \textcite{empowermentThroughFacebook}

Reviewer: ```suggestion
Ein Computer ist in Indien ein Luxusgut, welches durch die rasante Entwicklung von Mobilgeräten für die Jugend schnell an Bedeutung verlor, während Handys mit mobilem Internet allgegenwärtig wurden.
Durch die explosionsartige Verbreitung der neuen Kommunikationsmöglichkeiten entstanden in kürester Zeit große Gruppen im Internet, welche sich über neue Technologien, Wissenswertes und Ihre Interessen auszutauschen.
Dieser rapide Aufbau von Gesellschaften ist ein neues Phänomen, welches vorallem in `Entwicklungsländern' auftritt, in denen neue, und vergleichsweise preiswerte Technologien, den Zugang zum Internet ermöglichen.   \textcite{empowermentThroughFacebook}
```

Reviewer: `rapide Aufbau von Gesellschaften` -> `rapide Aufbau von gesellschaftlichen Strukturen`?

Reviewer: Wenn das Quelle Facebook, also Studie bezahlt von Facebook, das vlt. in den Text mit einfließen lassen.
Z.b.: `Laut einer von Facebook in Auftrag gegebenen Studie:`

Author: Ich glaube die Studie, welche ich hier benutzt habe, war nicht von Facebook in Auftrag gestellt worden, kann aber gerne den Satz ggf. nochmal umformulieren :)

\medskip

> Die durchschnittliche Bandbreite des mobilen Netzes war mit 2.8 Mb/s der durchschnittlichen festen Internetleitung überlegen, denn diese betrug lediglich 2.3 Mb/s\autocite{slideshareIndia}.

Reviewer: ```suggestion
Die durchschnittliche Bandbreite mobilen Internets war mit 2.8 Mb/s der durchschnittlichen Bandbreite eines Festanschlusses, von 2.3 Mb/s, überlegen \autocite{slideshareIndia}.
```

Reviewer: Gutes Beispiel bei dem die Fakten, anstatt deren Auswirkungen im Mittelpunkt stehen.
Diese puren Fakten könnten gut in einen Text eingepflegt werden, z.B.:

Ein Aspekt, weshalb die mobilen Endgeräte auch allgegenwärtig wurden, ist deren Verbindungsgeschwindigkeit, welche mit 2.8 Mb/s der eines Festanschlusses (2.3) überlegen ist.

---

Auf dem Land wird  WLAN übrigens auch über das mobil Netz betrieben, das es dort i.d.R. keine Kabelanschlüsse in die Häuser gibt.

> Da eine Prepaid Karte mit 1GB Datenvolum ca. 3.55\% des durchschnittlichen Monatsgehalt kosteten \autocite{broadbandAgency}, können sich hauptsächlich nur die Inder Internet leisten, welche in Stadtregionen leben, da dort das Gehalt weitaus höher ist.

Reviewer: ```suggestion
Da der Preis einer Prepaid Karte, mit 1 GB Datenvolumen, ca. 3.55\% des durchschnittlichen Monatsgehalt beträgt \autocite{broadbandAgency}, können sich hauptsächlich nur die Inder Internet leisten, welche in Stadtregionen leben, da dort das Einkommen weitaus höher ist.
```

>  Der Großteil der indischen Bevölkerung lebt aber im ländlichen Gebiet  und gerademal 31,2 \% \autocite{IndiaBevölkerung} in städtischen Gebieten. 

Reviewer: ```suggestion
 Der Großteil der indischen Bevölkerung lebt aber auf dem Land, und gerademal 31,2\% \autocite{IndiaBevölkerung} in städtischen Gebieten. 
```

> Von den 350 millionen aktiven Internetnutzern leben also lediglich 61 millionen nicht in einer Stadt. Somit benutzten 2015 gerademal 7.5 \%, auf dem Land lebende Inder, aktiv das Internet \autocite{slideshareIndia}.

Reviewer: Diesen Satz mit bitte mit dem darüber verbinden, z.B.:

```
Ein Großteil der Inder (69%) leben auf dem Land, sie machen aber nur 60 Mill der 350 Mill Internetnutzer Indien aus.
```

In die Richtung. Und alles noch ein bisschen mehr ineinander fließen lassen, sodass du nicht viele völlig entkoppelte Paragraphen hast.

Author: True. Werde das alles nochmal abrunden, nachdem ich 1.2.1 hochgeladen hab.

> Grund dafür ist vorallem die fehlende Abdeckung mobilen Internets in ländlichen Gebieten und der hohe Preis, was dazu führt, dass vorallem Jugendliche vom Land in städtische Gebiete ziehen, um dort bessere Bildung und bessere Arbeitsplätze zu beziehen und somit die ländlichen Gebiete noch mehr dem Armut verfallen. 

Reviewer: `, was dazu führt, dass vorallem Jugendliche` ließt sich als ob die Jungendlichen wegen des teuren Internets in die Städte ziehen :)

> %Überleitung zu Internet.org - > rural poor people + Internet.org = rural rich people?

Reviewer: Kurz darauf eingehen wie Internet.org diese Situation verbessern soll.

Reviewer: Also Teilnahme an der Informationsgesellschaft und am globalen Markt, etc.

\begin{document}

\maketitle
% 1.2.2 in der Gliederung
\subsubsection{Internetverbreitung}

> In Indien leben 2015 schon 350 Millionen Menschen\autocite{slideshareIndia}, welche das Internet aktiv nutzen, und davon sind 159 Millionen mobil per Handy im Internet unterwegs. Obwohl Indien damit schon den 2. Platz der aktiven Internetnutzer weltweit genießt, haben lediglich 27\% der indischen Bevölkerung Zugang zum Internet\autocite{InternetCountry}.

Reviewer: ```suggestion
In Indien leben 2015 schon 350 Millionen Menschen\autocite{slideshareIndia} welche das Internet aktiv nutzen, wovon allein 159 Millionen per Mobilgerät unterwegs sind. Obwohl Indien damit schon den 2. Platz der aktiven Internetnutzer weltweit genießt, haben lediglich 27\% der indischen Bevölkerung Zugang zum Internet\autocite{InternetCountry}.
```

\medskip 

> Ein Computer ist in Indien ein Luxusgut, welches durch die rasante Entwicklung von Mobilgeräten für die Jugend schnell an Bedeutung verlor, während Handys mit mobilem Internet allgegenwärtig wurden.

Reviewer: ```suggestion
Ein Computer ist in Indien ein Luxusgut, welches durch die rasante Entwicklung von mobilen Endgeräten für die Jugend schnell an Bedeutung verlor, während Handys mit mobilem Internet allgegenwärtig wurden.
```

> Durch die explosionsartige Verbreitung der neuen Kommunikationsmöglichkeiten entstanden in kürzester Zeit große Gruppen im Internet, welche sich über neue Technologien, Wissenswertes und Ihre Interessen austauschen.

Reviewer: ```suggestion
Durch die explosionsartige Verbreitung der neuen Kommunikationsmöglichkeiten entstanden in kürzester Zeit große Gruppen im Internet, welche sich über neue Technologien, Wissenswertes und Ihre Interessen austauschten.
```

> Dieser rapide Aufbau von Gesellschaften ist ein neues Phänomen, welches vorallem in `Entwicklungsländern' auftritt, in denen neue, und vergleichsweise preiswerte Technologien, den Zugang zum Internet ermöglichen.   \textcite{empowermentThroughFacebook}

Reviewer: ```suggestion
Dieser rapide Aufbau von gesellschaftlichen Strukturen ist ein neues Phänomen, welches vorallem in `Entwicklungsländern' auftritt, in denen neue, und vergleichsweise preiswerte Technologien, den Zugang zum Internet ermöglichen.   \textcite{empowermentThroughFacebook}
```
